includes/abstracts/abstract-wc-order.php

```diff
@@ -809,7 +808,7 @@ protected function get_items_key( $item ) {
 		} elseif ( is_a( $item, 'WC_Order_Item_Coupon' ) ) {
 			return 'coupon_lines';
 		} else {
-			return '';
+			return 'line_items';
```

Comment:
Would it be better to have a filter on the type vs always defaulting to a line_item? I'm worried if someone has a custom line item but doesn't set all required data, it will cause errors elsewhere.